pages/resources/testnet.md

```diff
@@ -10,8 +10,8 @@ import EVMAssets from '../../components/evm/assets'
 
 | Variable              | Value     |
 | --------------------- | --------- |
-| `axelar-core` version | `v0.26.5` |
-| `vald` version        | `v0.26.5` |
+| `axelar-core` version | `v0.28.0` |
```

Comment:
Looks like we missed this in the last upgrade

Reply:
yes, I probably missed it in the PR when doing testnet upgrade doc